src/components/GamePanel.vue

```diff
@@ -23,83 +30,72 @@
 import { defineComponent } from "vue";
 import MemoryCard from "./MemoryCard.vue";
 import PairItem from "./PairItem.vue";
-import { CardContent } from "../types/DataModels";
+import { CardData, CardPair } from "../types/DataModels";
+import { MemoryController } from "@/types/MemoryController";
+
 export default defineComponent({
```

Comment:
We use the script setup feature which is new in vue 3 (https://github.com/Gamify-IT/docs/blob/main/dev-manuals/languages/vue/styleguide.md). Would not be that much to refactor this.

Reply:
What do I have to change apart from changing <script> to <script setup>?